app/app.py

```diff
@@ -13,7 +13,10 @@ def coconut():
     """
     Handles Coconut code submitted by users.
     """
-    code = request.get_data(as_text=True)
+    code = request.values.get('code')
```

Comment:
We changed the endpoint from taking a json to text in 3e5839d4649c33a0bba0464448b613a640bb42d4. This makes `code = None` and we get the following error:
<img width="733" alt="screen shot 2018-03-02 at 1 26 23 pm" src="https://user-images.githubusercontent.com/8571671/36922735-dc4e0a82-1e1d-11e8-9d0c-2283ae9eaee6.png">

Reply:
What should I do to reproduce this error?

Reply:
Running locally (with frontend) on any input causes this error.

Reply:
Updated endpoint (see https://github.com/cs121-team-panda/coconut-interpreter-flask/commit/b845c82a117883dca78cdb56938d86e0b6bd162c), just pull master. We'll use `code = request.form['code']`.